### PR TITLE
appcode-bundled-jdk

### PR DESCRIPTION
**Pull request by yurikoles**

### CODE REVIEW

**Comment by fanquake** [comment]
The AppCode EAP is only available with the bundled JDK, so we don't a separate `Appcode EAP Bundled JDK` cask, nor do you need to add the Java 6 caveat to the EAP cask.

**Comment by yurikoles** [comment]
I have removed ```Appcode EAP Bundled JDK``` and rebased.

File: Casks/appcode-bundled-jdk.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                  '~/Library/Preferences/AppCode32',
+                  '~/Library/Application Support/AppCode32',
+                  '~/Library/Caches/AppCode32',
+                  '~/Library/Logs/AppCode32',
+                 ]
+
+  conflicts_with :cask => [
+                           'appcode',
+                           'appcode-eap',
+                           'appcode-eap-bundled-jdk',
```
Comment (fanquake) [inline Casks/appcode-bundled-jdk.rb]: Please remove this line

File: Casks/appcode-eap.rb [hunk 2]
```diff
@@ -16,4 +16,10 @@
                   '~/Library/Caches/AppCode33',
                   '~/Library/Logs/AppCode33',
                  ]
+
+  conflicts_with :cask => [
+                           'appcode',
+                           'appcode-bundled-jdk',
+                           'appcode-eap-bundled-jdk',
```
Comment (fanquake) [inline Casks/appcode-eap.rb]: Same here

**Comment by yurikoles** [comment]
@fanquake, done

**Comment by fanquake** [comment]
:beers:

**Pull request merged**